pulseaudio-control.bash

```diff
@@ -300,8 +320,16 @@ function output() {
 
 
 function usage() {
-    echo "Usage: $0 ACTION"
-    echo ""
+    echo "Usage: $0 [OPTION...] ACTION"
```

Comment:
I would actually prefer the options to be at the end for readability but I guess that's harder to implement, right?

Reply:
I don't think it should be harder, just different. :slightly_smiling_face: 

In front is just my own UX preference since they affect _all_ actions, e.g. I would do:
```
foo --config-dir=fooconfig bar --maximum-bar
```
where there's flags that apply both 'globally' and just for a given subcommand 'bar'.

Reply:
Yes that makes sense, but not in this script specifically because we can only have a single action at a time. What I was trying to say is that `foo --a-bunch-of-options action` makes it hard to find what action is being used (which is the most important part of the command), compared to `foo action --a-bunch-of-options`. But we can keep your way just for consistency with other scripts and It's ok.